js/main.js

```diff
@@ -108,7 +133,7 @@ var init_hvac = function () {
 function setup_ui() {
     console.log("setup_ui() called!");
 	$(".noUiSliderLeft").noUiSlider({
-	    range : [ 0, 14 ],
+	    range : [ -1, 14 ],
```

Comment:
I have minor concerns about the range starting at -1. But if the behavior is what we want, then it should be fine. We should research into why this works like this. IIRC, the mobile client uses the same library and does not have this issue (but I may be wrong)